File: app/src/main/java/dev/ragnarok/fenrir/api/model/upload/UploadPhotoToMessageDto.java
package dev.ragnarok.fenrir.api.model.upload;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public class UploadPhotoToMessageDto {

    @SerializedName("server")
    public int server;

    @Nullable
    @SerializedName("photo")
    public String photo;

    @Nullable
    @SerializedName("hash")
    public String hash;

    @NonNull
    @Override
    public String toString() {
        return "UploadPhotoToMessageDto{" +
                "server=" + server +
                ", photo='" + photo + '\'' +
                ", hash='" + hash + '\'' +
                '}';
    }
}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/api/model/upload/UploadPhotoToMessageDto.kt
package dev.ragnarok.fenrir.api.model.upload

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class UploadPhotoToMessageDto {
    @SerialName("server")
    var server = 0L

    @SerialName("photo")
    var photo: String? = null

    @SerialName("hash")
    var hash: String? = null
    override fun toString(): String {
        return "UploadPhotoToMessageDto{" +
                "server=" + server +
                ", photo='" + photo + '\'' +
                ", hash='" + hash + '\'' +
                '}'
    }
}
